kolibri/core/discovery/test/test_network_utils.py

```diff
@@ -301,6 +302,14 @@ def test_request__response_failure(self):
                     with NetworkClient("http://sadurl.qqq") as nc:
                         nc.connect()
 
+    @mock.patch.object(requests.Session, "get", mock_response(200))
+    def test_request__user_agent(self):
+        client = NetworkClient()
+        response = client.get("https://example.com")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue("kolibri/{0}".format(kolibri.__version__) in client.headers["User-Agent"])
```

Comment:
This might fail? It seems `get_user_agent` returns `Kolibri` with a capital `K` whereas this is checking lowercase `k`?

Reply:
Thanks, sorry didn't notice that my test failed to run locally. Pushed a fix and confirmed it's passing locally.